### ISSUE TITLE
BB10-WebWorks-Samples / Toast dont execute a dissmissCallback

### ISSUE THREAD

**Issue by theclabs** [blackberry/BB10-WebWorks-Samples#158]
On disappear toast, dont call te dissmissCallback function

https://github.com/blackberry/BB10-WebWorks-Samples/blob/master/Toast/js/app.js#L50

**Comment by 0xLuz** [comment]
Do you are using the bbUI.js/bbUI.css version 0.9.6? If not, try to update https://www.dropbox.com/s/kvbef82rv4b5ak8/bbui.zip
I was having some problems to call functions with the old version, the most samples here, doesn't are updated.

**Comment by gcsantiago** [comment]
You need to call with button on it for first time to ensure to work (bug??)

https://github.com/blackberry/BB10-WebWorks-Samples/issues/66

**Comment by timwindsor** [comment]
Is that still happening on the latest OS or developer preview OS? The original issue suggests that it should be fixed by now.

**Comment by theclabs** [comment]
@gcsantiago yes, have a button in the function, but dont execute anyway. look ( https://github.com/blackberry/BB10-WebWorks-Samples/blob/master/Toast/js/app.js#L50 ) 
@timwindsor: yes happening in the last simulators and 10.2 final releases.

**Comment by jeffheifetz** [comment]
Is this issue specifically with the sample, or with the API?

**Comment by timwindsor** [comment]
@ctetreault is the author, so perhaps he can take a look. We are updating some of these samples shortly anyway.

**Comment by ctetreault** [comment]
Pretty sure it's just the sample Jeff. I use toasts with callbacks in the upcoming BFB Boilerplate without a problem.

We're updating the old samples shortly to correct a few issues, I'll make sure this one is on the list.

**Comment by T-M-C** [comment]
Every time my app shows a toast (with no dismisscallback), i get an error in console:
`TypeError: 'undefined' is not an object (evaluating 'options.dismissCallback')`
